rcav2/logjuicer.py

```diff
@@ -98,6 +98,24 @@ async def get_remote_report(env: Env, url: str, worker: None | Worker) -> Report
             env.log.error("WS error :/", e)
 
 
+async def get_report_from_file(env: Env, file_path: str) -> Report:
```

Comment:
nit: this method is not async, it's surprising that the linter doesn't warn about it.